app/helpers/application_helper/button/basic.rb

```diff
@@ -12,6 +12,11 @@ def initialize(view_context, view_binding, instance_data, props)
     end
   end
 
+  def check_rbac?
+    return true if self[:child_id].nil?
+    role_allows?(:feature => self[:child_id])
```

Comment:
@jrafanie : yes, it's a RBAC feature identifier and also a button id. In this case it's an id of a button under a drop-down button.

The change you are speaking about is out of scope for this PR. However this PR together with the other "button RBAC" PRs will limit the number of places that will have to be changed as you wrote.